sites/skeleton.dev/src/components/docs/Drawer.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from 'react';
```

Comment:
I'd recommend adding the following:

Add this for creating a slide left/right animation with CSS via `translateX()`:
https://floating-ui.com/docs/useTransition

Use this to add a backdrop overlay to darken the background:
https://floating-ui.com/docs/FloatingOverlay